config/pre-install/v0.18.0/cloudauditlogssource.yaml

```diff
@@ -0,0 +1,218 @@
+                properties:
+                  name:
+                    type: string
+                  key:
+                    type: string
+                  optional:
+                    type: boolean
+              project:
+                type: string
+                description: >
+                  Google Cloud Project ID of the project into which the topic should be created. If omitted uses
+                  the Project ID from the GKE cluster metadata service.
+              serviceName:
+                type: string
+              methodName:
+                type: string
+              resourceName:
+                type: string
+          status: &status
+            type: object
+            properties: &statusProperties
+              observedGeneration:
+                type: integer
+                format: int64
+              conditions:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    lastTransitionTime:
+                      # We use a string in the stored object but a wrapper object at runtime.
+                      type: string
+                    message:
+                      type: string
+                    reason:
+                      type: string
+                    severity:
+                      type: string
+                    status:
+                      type: string
+                    type:
+                      type: string
+                  required:
+                    - type
+                    - status
+              sinkUri:
+                type: string
+              ceAttributes:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                    source:
+                      type: string
+              projectId:
+                type: string
+              topicId:
+                type: string
+              subscriptionId:
+                type: string
+              stackdriverSink:
+                type: string
+                description: >
+                  ID of the Stackdriver sink used to publish audit log messages.
+  - <<: *version
+    name: v1alpha1
+    # TODO: Flip served bit of v1alpha1 in https://github.com/google/knative-gcp/issues/1544.
+    served: true
+    storage: false
+    # v1alpha1 and v1beta have status.properties.serviceAccountName in the schema
+    schema: &v1alpha1Schema
+      openAPIV3Schema:
+        <<: *openAPIV3Schema
+        properties:
+          <<: *properties
+          status:
+            <<: *status
+            properties:
+              <<: *statusProperties
+              serviceAccountName:
+                type: string
+  - <<: *version
+    name: v1beta1
+    served: true
+    storage: false
+    # the schema of v1beta1 is exactly the same as v1alpha1 schema
+    schema: *v1alpha1Schema
```

Comment:
Since we are gonna remove v1alpha1 support, this will get deleted in separate PR, right?

Reply:
Yes. Please see the comments I added.